Formula/infer.rb

```diff
@@ -15,12 +15,21 @@ class Infer < Formula
 
   depends_on "autoconf" => :build
   depends_on "automake" => :build
+  depends_on "camlp4" => :build
   depends_on "cmake" => :build
+  depends_on "gnu-sed" => :build
   depends_on :java => ["1.7+", :build]
   depends_on "libtool" => :build
   depends_on "ocaml" => :build
   depends_on "opam" => :build
+  depends_on "perl" => :build
   depends_on "pkg-config" => :build
+  depends_on :x11 => :build
```

Comment:
This should be removed.

Reply:
I agree this should be removed; from our conversations in the earlier PR though it's beyond my knowledge how to remove it. Will need someone more experienced with ocaml to take a look and come up with the proper procedure of `pin add`ing the ocaml package and adding the `-no-graph` flag to its configure step (sounds simple enough but I could not get it to work).